spec/dummy/spec/fixtures/csv_import/csv_rearranged_headers_new.csv

```diff
@@ -0,0 +1,2 @@
+title*, creator*, keyword*, rights_statement*,files*,deduplication_key*, description, resource type, license, subject  
+the great gatsby, f scott fitz, yearning, undefined, tgg.pdf, tgg-fscott, novel by f. scott fitz, unknown, american literature
```

Comment:
I don't see where this fixture gets used in a test.  It feels like we should have tests that confirms the headers with an asterisk and the same headers without the asterisks get mapped correctly.